client/python/armada_client/gen/event_typings.py

```diff
@@ -10,7 +10,16 @@ def get_event_states():
 
 
 def get_all_job_event_classes():
```

Comment:
Is this backwards compatible?

And can you update airflow operator grpc dependency?

Reply:
Define backwards compatible? Will this fail if we lower the version down? I am not sure, I'll test it soon